apps/hpe-csi/data.yaml

```diff
@@ -0,0 +1,39 @@
+tags:
+  - Storage
+  - Drivers and plugins
+title: "HPE CSI Driver"
+summary: "TODO"
```

Comment:
HPE CSI Driver for Kubernetes is a multi-vendor and multi-platform Container Storage Interface driver.

Reply:
Ok, thank you

Reply:
@datamattsson it's longer than 90 chars limit. Can it be: "Multi-vendor and multi-platform Container Storage Interface driver." ?